### ISSUE TITLE
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.hljs-comment:nth-child(2)): A11y_WCP URLs - ONNX Runtime_ONNX Runtime_Home_AI4W.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20153]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; A11yWCAG2.1; #A11yMAS; #color-contrast; #WCAG1.4.3; #A11yAuto;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2
### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Run Accessibility Insights for web.
4. Now check that Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.hljs-comment:nth-child(2))
### Issue:
 Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)
### Target application: 
ONNX Runtime | Home - https://onnxruntime.ai/
### Element path: 
.hljs-comment:nth-child(2)
### Snippet: 
<span class="hljs-comment">// Load the model and create InferenceSession</span>
### Related paths: 
  pre[data-language="cpp"] > code
### How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 2.46 (foreground color: #a0a1a7, background color: #fafafa, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
###  Microsoft Edge version 123.0.0.0
====
This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Attachment:

![Screenshot 2024-03-30 153737](https://github.com/microsoft/onnxruntime/assets/142479541/531b2fe9-950a-4065-ab2b-63515b318a3d)

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/training
Edge version:125.0.2535.92 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
For reference attachment is added.

#Regressed:06-17-24; #Closed;

### Attachment:

![image](https://github.com/microsoft/onnxruntime/assets/142479541/d61d9128-3d8d-4f1d-89a8-5cb9b4666645)
